Dockerfile.const

```diff
@@ -34,6 +33,14 @@ COPY scripts ./scripts
 COPY ethereum/.env.test ./ethereum/.env.test
 COPY alephium/.env.test ./alephium/.env.test
 
+ENV NUM_GUARDIANS=$num_guardians
+# WARNING:
+# Credentials file should only be used when generating VAAs for testnet and mainnet, it
+# should never be checked in!
+COPY README.md sensitive/credentials-file.json* .
```

Comment:
Does this mean that we will put the credential files in the docker image?

Reply:
It was used to output the env file locally, which is later used to build docker images. But as we discussed the other day, my understanding was off and this is not needed any more.